packages/clay-css/src/scss/variables/_empty-state.scss

```diff
@@ -13,6 +13,19 @@ $c-empty-state-animation: map-deep-merge(
 	$c-empty-state-animation
 );
 
+$c-empty-state-animation-sm: () !default;
+$c-empty-state-animation-sm: map-deep-merge(
+	(
+		align-items: center,
```

Comment:
A lot of styles are duplicated, we can just inherit them from the original component.

Reply:
@pat270 could you please have another look, it seems @sergiojimcos updated this. Thanks!